lib/taskana-core/src/main/resources/sql/taskana-schema.sql

```diff
@@ -63,6 +62,7 @@ CREATE TABLE TASK (
         PLANNED TIMESTAMP NULL,
         DUE TIMESTAMP NULL,
         NAME VARCHAR(255) NULL,
+        CREATOR VARCHAR(255) NULL,
```

Comment:
please change to VARCHAR(32). That should be fine for just userid (no group names!)